chatmd/importers/chatgpt.py

```
@@ -0,0 +1,387 @@
+
+@dataclass
+class _Conversation:
+    data: Dict
+    path: Path
+
+
+def _load_export(path: Path) -> Tuple[Path, Optional[TemporaryDirectory]]:
+    if path.is_dir():
+        return path, None
+    if path.suffix.lower() != ".zip":
+        raise FileNotFoundError(f"Unsupported ChatGPT export: {path}")
```

Ruff (0.15.9)

[warning] Avoid specifying long messages outside the exception class

(TRY003)